pages/platform/ai/deploy_tuto_10_locust/guide.en-gb.md

```diff
@@ -0,0 +1,252 @@
+
+For this tutorial, we will add 480 users in total, and a spawn rate of 2 users added per second. We will simulate this for a duration of 4 minutes. We suppose that this case is for a rush on the API. Most of the time, we can assume that there are not so much users on the platform. 
+
+Launch the test. Tic, Tac, the results will be in the next part! 
+
+### Interpret the results via Locust
+
+At the end of the load test, you will see this quick summary: 
+
+![image](images/result_locust.png){.thumbnail}
+
+If we want to get more details about our test, we can see the graphs provided by Locust in the tab `charts`. Here is what we can see:
+
+![image](images/locust_graphs.png){.thumbnail}
+
+
+We deployed this API from 1 to 5 replicas, with 1 CPU for each of them, completed with auto-scaling. We can see that our API has been a bit overloaded. Indeed, with the graphics or with the quick summary, we can see some failures. These failures are due to server error. We may suppose  that we reached some hardware limits at some point, for example before scaling to a greater amount of replicas.
+
+Also, we can see an increasing call latency over time. At the end, we can expect more than 3 seconds per call.
+
+Result interpretation may depend of your needs and performance criterias. We can of course make a new test with more users to see the limits of our APIs, and put several tasks in the `locustfile.py`. 
+
+One thing cannot be seen here: OVHcloud backend scaling. We deployed our app with autoscaling, from 1 replica minimum to 3 replicas maximum.
+Did we use them ? Were they useful and at maximum capacity?
+
+Let's see the same results in details with the AI Deploy monitoring tool.
+
+### Interpret the results with the AI Deploy Monitoring
+
+Go in OVHcloud control panel, and get the detail of your deployed application. Click on the `Access Dashboards`button.
+
+
+This dashboard is provided for free in AI Deploy, for each deployed application. All of the deployed apps are combined in a simple Grafana Dashboard.
+
+You can select the deployed app on the top of this Grafana dashboard, as shown below: 
+
+![image](images/general_dashboard.png){.thumbnail}
+
+With this dashboard, you can see the percentage of CPU used in real time, the HTTP latency of your API, the autoscaling of the app, network bandwidth and more. 
+Vertical blue bar shows scaling events.
+
+Here is the result for the CPU load, overall (all replicas combined): 
+
+![image](images/cpu_spam.png){.thumbnail}
+
+We can see that our app has scaled few times hasn't reach maximum capacity usage, thanks to autoscaling. Let's now take a look at the latency of our application:
+
+![image](images/latency_spam.png){.thumbnail}
+
+Here we see that the latency has increased  gradually, since we hava a spawn rate of two new user per second. API latency was stable with approximately 750ms then peaked to 1.5s. 
```

Comment:
```suggestion
Here we see that the latency has increased gradually, since we have a spawn rate of two new user per second. API latency was stable with approximately 750ms then peaked to 1.5s. 
```